src/mono/wasm/debugger/DebuggerTestSuite/SteppingTests.cs

```diff
@@ -1037,5 +1037,36 @@ public async Task DebuggerHiddenIgnoreStepUserBreakpoint(string steppingFunction
                 step_into2["callFrames"][0]["location"]["lineNumber"].Value<int>()
                 );
         }
+
+        [ConditionalTheory(nameof(RunningOnChrome))]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task SteppingIntoLibrarySymbolsLoadedFromSymbolServer(bool justMyCode)
```

Comment:
Maybe also:

- a test that first hits the bp, and steps with no symbol server set. Then in the same test, sets the symbol server, and  steps again
- A separate test that (a) sets a symbol server; (b) steps; (c) adds another symbol server; (d) steps again
- A separate test that (a) adds multiple symbol servers; (b) steps; (c) removes one of the symbol servers; (d) steps again